assets-sync.js

```diff
@@ -498,19 +498,19 @@
     async buildLocalInventory(referenceDirs) {
         referenceDirs = referenceDirs || new Set();
         // Add the root dir to the reference list
-        referenceDirs.add(this.apiKeyPath ? this.apiKeyPath : "/");
+        referenceDirs.add(ForgeAssetSync.sanitizePath(this.apiKeyPath ? this.apiKeyPath : "/"));
```

Comment:
This is necessary because an API key which shares a specific folder adds that base folder path to the local inventory here.
![image](https://github.com/user-attachments/assets/a8025aae-4b35-4285-8699-845946f852fe)

If that folder contains a character that needs to be sanitized, e.g. `test:folder`, then it should get sanitized before it gets added to the reference list, so that it does not cause an error when the sync attempts to create that folder.